tests/utils/paper_helpers.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+def compare_formatted_citations(expected: str, actual: str) -> bool:
+    """
+    Compares two formatted citation strings; ignoring the citation_count value.
+
+    :param expected: The expected formatted citation string.
```

Comment:
We use Google Style docstrings, you'll perhaps want to update your IDE

Reply:
No longer required